packages/ui/src/components/grid/components/container/container.styles.ts

```diff
@@ -2,7 +2,7 @@ import { tv } from 'tailwind-variants';
 
 export const styles = tv(
   {
-    base: 'mx-auto box-border w-full max-w-container px-2 xsl:px-5 sm:px-6 md:px-8 lg:px-10',
+    base: 'mx-auto box-border w-full max-w-container px-4 xsl:px-5 sm:px-6 md:px-8 lg:px-10',
```

Comment:
@jaortiz is this change going to collide with grid changes you are working?

Reply:
Yes it will as current GEL grid has default padding-x of 12px. @kenjishiromajp are you not able to just override that padding for your use case?